en/02_Developer_Guides/18_Cookies_And_Sessions/02_Sessions.md

```diff
@@ -100,9 +100,52 @@ including form and page comment information. None of this is vital but `clear_al
 $session->clearAll();
 ```
 
-## Cookies
+## Configuration
 
-### Samesite attribute
+### Session lifetime
+
+By default, the session cookie will only last until the user closes their browser, and the session on the server-side will live for the number of seconds set in the [`session.gc_maxlifetime`](https://www.php.net/manual/en/session.configuration.php#ini.session.gc-maxlifetime) ini configuration since it was last touched.
+
+You can configure the session lifetime by setting the [`Session.timeout`](SilverStripe\Control\Session->timeout) configuration property to the number of seconds the session should be valid for since the last modified time.
+
+```yml
+SilverStripe\Control\Session:
+  timeout: 86400
+```
+
+> [!WARNING]
+> If you use a save handler which isn't included in `silverstripe/framework`, it may treat the session lifetime differently. You should check its docuemntation or implementation to understand how it works.
```

Comment:
```suggestion
> If you use a save handler which isn't included in `silverstripe/framework`, it may treat the session lifetime differently. You should check its documentation or implementation to understand how it works.
```

Reply:
Done